### ISSUE TITLE
Javascript template

### ISSUE THREAD

**Issue by edap** [micknoise/Maxim#13]
Is it the javascript template supposed to work only with processing js? I've followed your course online about audiovisual content creation, and I wanted to use maxim js. Unfortunately I was not able to integrate it in my current setup (es6 + webpack). A simple javascript example without processing js would be nice, if available. Thanks for your work ;) I've liked the course this summer and the c++ library

**Comment by admin96** [comment]
I don't believe it is necessary to use processing.js at all. The maxim library is built on top of the Web Audio API and processing.js is basically interfacing with the HTML canvas. The two can definitely function independent of one another. You can find maxim.js examples here: https://live.codecircle.com/

**Comment by edap** [comment]
I know, I was taking part at that course. But as it is now maxim.js can not be integrated as an es6 module, at least this was my conclusion after trying to use it in a project with webpack and es6.
